### ISSUE TITLE
false positives for cacti

### ISSUE THREAD

**Issue by Jibec** [WeblateOrg/translation-finder#52]
Here are a list of results that may come from the same issue as #51 
My guess is that "to" is considered as a language code, which makes "How-to-Data-Query-Templates.html" considered as a translation file.
But if there isn't at least two languages codes detected, it's most probably a false positive. 

```
[ { 'file_format': 'html',
    'filemask': 'docs/How-*-Data-Query-Templates.html'},
  { 'file_format': 'html',
    'filemask': 'docs/How-*-Determine-Template-Version.html'},
  { 'file_format': 'html',
    'filemask': 'docs/How-*-Graph-a-Custom-Collection-Script.html'},
  { 'file_format': 'html',
    'filemask': 'docs/How-*-Graph-Your-Network.html'},
  { 'file_format': 'html',
    'filemask': 'docs/How-*-Setup-Remote-Pollers.html'},
  { 'file_format': 'html',
    'filemask': 'docs/How-*-SSH-Tunnels.html'},
  { 'file_format': 'json-nested',
    'filemask': 'include/vendor/cldr-to-gettext-plural-rules/src/cldr-data/main/*/languages.json'},
  { 'file_format': 'json-nested',
    'filemask': 'include/vendor/cldr-to-gettext-plural-rules/src/cldr-data/main/*/scripts.json'},
  { 'file_format': 'json-nested',
    'filemask': 'include/vendor/cldr-to-gettext-plural-rules/src/cldr-data/main/*/territories.json'},
  { 'file_format': 'php',
    'filemask': 'cli/sqltable_*_php.php'},
  { 'file_format': 'php',
    'filemask': 'include/*/css/index.php'},
  { 'file_format': 'php',
    'filemask': 'include/*/index.php',
    'template': 'include/index.php'},
  { 'file_format': 'php',
    'filemask': 'include/*/less/index.php'},
  { 'file_format': 'php',
    'filemask': 'include/*/scss/index.php'},
  { 'file_format': 'php',
    'filemask': 'include/*/webfonts/index.php'},
  { 'file_format': 'php',
    'filemask': 'include/vendor/gettext/src/Extractors/*.php'},
  { 'file_format': 'php',
    'filemask': 'include/vendor/gettext/src/Generators/*.php'},
  { 'file_format': 'php',
    'filemask': 'include/vendor/phpmailer/language/phpmailer.lang-*.php'},
  { 'file_format': 'php',
    'filemask': 'scripts/*_count_oids.php'},
  { 'file_format': 'php',
    'filemask': 'scripts/*_cpoller.php'},
  { 'file_format': 'php',
    'filemask': 'scripts/*_fping.php'},
  { 'file_format': 'php',
    'filemask': 'scripts/*_gexport.php'},
  { 'file_format': 'php',
    'filemask': 'scripts/*_host_cpu.php'},
  { 'file_format': 'php',
    'filemask': 'scripts/*_host_disk.php'},
  { 'file_format': 'php',
    'filemask': 'scripts/*_hstats.php'},
  { 'file_format': 'php',
    'filemask': 'scripts/*_net_snmp_disk_bytes.php'},
  { 'file_format': 'php',
    'filemask': 'scripts/*_net_snmp_disk_io.php'},
  { 'file_format': 'php',
    'filemask': 'scripts/*_poller.php'},
  { 'file_format': 'php',
    'filemask': 'scripts/*_sql.php'},
  { 'file_format': 'php',
    'filemask': 'scripts/*_webseer.php'},
  { 'file_format': 'po',
    'filemask': 'locales/po/*.po',
    'new_base': 'locales/po/cacti.pot'}]
```

file is inside https://kojipkgs.fedoraproject.org//packages/cacti/1.2.14/1.fc32/src/cacti-1.2.14-1.fc32.src.rpm
[list of files](https://github.com/WeblateOrg/translation-finder/files/5308326/hop.txt)

شكر (I understand this means thanks in Arabic)

**Comment by nijel** [comment]
Should be fixed by 6aabffff01247862b7d7aa2e971a829d5423deef

**Comment by Jibec** [comment]
![image](https://user-images.githubusercontent.com/474883/95254433-310f6400-0820-11eb-9cc8-1b34de8557c6.png)

I think lang codes "fa" and "ss" still trigger many false positives and suggest to add a thresholds. Not sure we can do much here.
